packages/babel-core/src/config/files/configuration.ts

```diff
@@ -263,6 +263,8 @@ const readConfigJSON5 = makeStaticFileCache((filepath, content): ConfigFile => {
     throw err;
   }
 
+  delete options["$schema"];
```

Comment:
These should be moved after all the following `if` checks. Otherwise if, for example, `options.json` contains `null` this will throw a bad error rather than the `No config detected` one.

Reply:
Okay, I got that I will move it down.